eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/GovSpec.hs

```diff
@@ -41,8 +46,8 @@ import Test.Cardano.Ledger.Imp.Common
 spec ::
   forall era.
   ( ConwayEraImp era
-  , GovState era ~ ConwayGovState era
   , Inject (ConwayGovPredFailure era) (PredicateFailure (EraRule "LEDGER" era))
+  , era ~ Conway
```

Comment:
I think it'd be better to just replace `era` with Conway

Reply:
But it's better if you can get rid of it of course

Reply:
I tried, but there seem to be multiple rabbit holes to go through. :smile:

Reply:
Well, we need to jump through them. Because the tests that we are writing now, must be usable for future eras.

Reply:
Type application at a single place eradicated the need for these constraints. :bow: :raised_hands: . Thank you very much!